source/codegen/metadata_mutation.py

```diff
@@ -126,16 +126,17 @@ def populate_grpc_types(parameters, config):
     service_class_prefix = config["service_class_prefix"]
     for parameter in parameters:
         if 'grpc_type' in parameter:
-            pass
-        elif 'type_to_grpc_type' in config:
+          continue
+        if 'type_to_grpc_type' in config:
           type_map = config['type_to_grpc_type']
-          parameter['grpc_type'] = type_map[parameter['type']]
-        else:
-          is_array = common_helpers.is_array(parameter["type"])
-          parameter['grpc_type'] = get_grpc_type_from_ivi(
-              parameter['type'],
-              is_array,
-              service_class_prefix)
+          if parameter['type'] in type_map:
+            parameter['grpc_type'] = type_map[parameter['type']]
+            continue
```

Comment:
note: I refactored this code a little to try to do the lookup from `type_to_grpc_type`, but if that fails fall back on the other method.